src/manifolds/MetricManifold.jl

```diff
@@ -111,7 +234,7 @@ Return the [`LeviCivitaConnection`](@ref) for a metric manifold.
 connection(::MetricManifold) = LeviCivitaConnection()
 
 @doc raw"""
-    det_local_metric(M::AbstractManifold, p, B::AbstractBasis)
+    det_local_metric(M::AbstractManifold, p, B::AbstractBasis=DefaultOrthogonalBasis)
```

Comment:
I'd leave it for https://github.com/JuliaManifolds/Manifolds.jl/issues/424

Reply:
Yes sorry, can artefact from when I thought I might do that here and changed the doctoring already.